test/cmdlineTests/yul_source_locations/output.json

```diff
@@ -325,6 +325,7 @@ object \"C_54\" {
                 sum := add(x, y)
             }
 
+            /// @ast-id 30
```

Comment:
It seems that only `--ir` output was affected in tests. There are no `@ast-id` annotations added in `--ir-optimized`. I think that's because overall very few tests enable `--ir-optimized`. But still, from `AsmPrinter::formatDebugData()` I'd expect basically every `@src` comment in optimized output to get the `@ast-id` annotation and this does not seem to be the case. Why?

Can you enable `--ir-optimized` in this test? Also, it's a Standard JSON test so I think it should be named `standard_yul_source_locations`. I actually had to do this in my own PR to be able to create a non-JSON equivalent.

Reply:
It seems the optimizer prunes most of the ast ids, I can investigate a little if we can prevent it, but I think this was to be expected.

Reply:
Oh, this is actually due to the fact that the ast-id parsing is not implemented in this PR, so the optimized will always remove it.

Reply:
Ok, then it's fine I guess.